app/routes.test.js

```diff
@@ -295,10 +294,9 @@ describe('routes', () => {
         })
         .expect(200)
         .then((res) => {
-          expect(res.text).toEqual('Error adding user');
-          // expect(res.text.includes('')).toEqual(true);
-        // expect(res.text.includes('data-test-id="error-page-title"')).toEqual(false);
-        // addUserContext.getAddUserPageErrorContext.mockReset();
+          expect(res.text.includes('data-test-id="add-user-page"')).toEqual(true);
```

Comment:
Can we have an additional expect here just to check that it's the add-user-page but with errors?